nbsite/cmd.py

```diff
@@ -96,7 +97,7 @@ def build(what='html',
         for path in glob.glob(os.path.join(paths['doc'], '**', '*.ipynb'), recursive=True):
             print('Removing evaluated notebook from {}'.format(path))
             os.remove(path)
-    if _pyodide_enabled(os.path.join(paths['doc'], 'conf.py')):
+    if sys.platform == 'win32' and _pyodide_enabled(os.path.join(paths['doc'], 'conf.py')):
```

Comment:
We could look into [portalocker](https://pypi.org/project/portalocker/)

Reply:
No strong opinion.

Reply:
I think I'll use this.